src/analysis/env_lookup.ml

```diff
@@ -126,15 +129,26 @@ let by_longident (nss : Namespace.inferred list) ident env =
             let path, val_desc = Env.find_value_by_name ident env in
             raise
               (Found (path, Value, val_desc.val_uid, val_desc.Types.val_loc))
-          | `This_label lbl ->
+          | `This_label (Namespace.P lbl) ->
             log ~title:"lookup"
               "got label, fetching path and loc in type namespace";
             let path, loc = path_and_loc_from_label lbl env in
             let path = Path.Pdot (path, lbl.lbl_name) in
             raise (Found (path, Label, lbl.lbl_uid, loc))
           | `Labels ->
             log ~title:"lookup" "lookup in label namespace";
-            let lbl = Env.find_label_by_name ident env in
+            let module Pack = struct
+              (* This gadt is used to mint the existential type [rep] *)
+              type t = P : 'rep Types.gen_label_description -> t
```

Comment:
Why can't you use `packed_label_description` defined above?

Reply:
You're right, I can. Fixed